### PR TITLE
Update function and more python notebook

### PR DESCRIPTION
**Pull request by aguspesce**
Add mor formative assessment and improve text

### CODE REVIEW

**Comment by mdpiper** [comment]
This is a long lesson. Would there be room in our schedule to break it in two?

**Comment by aguspesce** [comment]
@mdpiper Do you think?
We only see what it is, setup, make a repo and connect it with Github.
The part about collaborative work is not included

**Comment by aguspesce** [comment]
@mdpiper 
Last season, we could teach the lesson on time.
Any way, we have space to do it.

I will practice this lesson and see how much time it takes me.

**Comment by mdpiper** [comment]
> Last season, we could teach the lesson on time.

Oh, OK--I defer to your experience, @aguspesce. I withdraw my comment!

**Comment by aguspesce** [comment]
it's ok! Your concern is correct.
let me see how much time it is

File: 06_loops_if_functions.ipynb [hunk 1]
```diff
@@ -4,20 +4,21 @@
    "cell_type": "markdown",
```
Comment (isamarcortes) [inline 06_loops_if_functions.ipynb:4]: <p>Ya que tenemos impares definido, podemos escribir el codigo como: </p><p> </p><p>for impar in impares:</p><p>print(impar)</p><p> </p><p>Así hacen la conexion con lo de arriba. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (isamarcortes) [inline 06_loops_if_functions.ipynb:4]: <p>Esto lo vamos a separar al google doc, cierto?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (isamarcortes) [inline 06_loops_if_functions.ipynb:4]: <p>Esto va al documento, cierto?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (isamarcortes) [inline 06_loops_if_functions.ipynb:4]: <p>Sounds like a formative assessment. Is it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CodeToCommunicate/CoCoLessons/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Pull request merged**